common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -307,14 +307,14 @@ export class SendTransaction extends React.Component<Props, State> {
                     onChange={readOnly ? void 0 : this.onGasChange}
                   />
                   {(offline || forceOffline) && (
-                    <div>
-                      <NonceField
-                        value={nonce}
-                        onChange={this.onNonceChange}
-                        placeholder={'0'}
-                      />
-                    </div>
-                  )}
+                      <div>
```

Comment:
I see no changes here except for formatting.

Reply:
I just added a new commit to clean up the rest of the formatting noise, the one thing I couldn't change was the indentation you highlighted, it gets formatted that way by the `precommit` script in `package.json`.